server/examples/config-ldes.json

```diff
@@ -88,7 +88,7 @@
                   "prefix": "indexed",
                   "view": {
                     "@type": "MongoSDSView",
-                    "descriptionId": "http://localhost:3000/ldes/#timestampFragmentation",
+                    "descriptionId": "http://127.0.0.1:3000/ldes/#timestampFragmentation",
```

Comment:
Same comment as previous so I don't think it makes sense to change it to 127.0.0.1.